src/api/app/services/webui_controller_service/user_checker.rb

```diff
@@ -1,17 +1,18 @@
 module WebuiControllerService
   class UserChecker
-    attr_reader :user_login, :http_request
+    attr_reader :user_login, :user_email, :http_request
 
     def initialize(http_request:)
       @http_request = http_request
-      @user_login = extract_user_login_from_http_request
+      @user_login = http_request.env['HTTP_X_USERNAME']
+      @user_email = http_request.env['HTTP_X_EMAIL']
```

Comment:
Can you just get rid of those and use the request object directly? There is no real cost associated to doing `http_request.env['HTTP_X_USERNAME']`